public/js/components/Table/PaginatedTable.js

```diff
@@ -0,0 +1,103 @@
+import React from "react";
+import Table from "./Table";
+import TablePagination from "./TablePagination";
+
+class PaginatedTable extends React.Component {
+    state = {
+        pageIndex: 0
+    };
+
+    get rowCount() {
+        return React.Children.count(this.props.children);
+    }
+
+    get pageCount() {
+        return Math.ceil(this.rowCount / this.props.pageSize);
+    }
+
+    get prevDisabled() {
+        return this.state.pageIndex < 1;
+    }
+
+    get nextDisabled() {
+        return this.state.pageIndex > this.pageCount - 2;
+    }
+
+    get visiblePageIndices() {
+        const { pagesAround } = this.props;
```

Comment:
Can you remind me what the `{}` does in this context?

Reply:
It's equivalent to `const pagesAround = this.props.pagesAround;`

Reply:
👍